drivers/usb_c/tcpc/Kconfig.tcpc_tcpci

```diff
@@ -31,3 +31,27 @@ config USBC_TCPC_TCPCI_TRANSMIT_RETRIES
 	  Maximum number of packet retransmissions done by TCPC
 
 endif
+
+config USBC_TCPC_PS8XXX
+	bool "USB-C TCPC device controller driver"
+	select USBC_TCPC_TCPCI
+	default y
+	depends on DT_HAS_PARADE_PS8XXX_ENABLED
+	help
+	  Enable USB-C TCPC support for the Parade PS8xxx chip family
+
+if USBC_TCPC_PS8XXX
+
+config USBC_TCPC_PS8XXX_INIT_DELAY
+	int "PS8xxx init delay"
+	default 25
+	help
+	  Delay between each try of the TCPC initialization
+
+config USBC_TCPC_PS8XXX_INIT_RETRIES
+	int "PS8xxx init retries"
+	default 10
+	help
+	  Number of initialization tries that will be performed
+
+endif
```

Comment:
Each driver should have it's own Kconfig file.

Reply:
Done.